scripts/mesh-point/mesh

```diff
@@ -1,19 +1,37 @@
-#!/bin/sh
+#!/bin/bash
+
+function isMeshable {
```

Comment:
Add doc on what the checks are trying to accomplish. Why the `'xradio_wlan'` check -> `result=''` is unclear to me.

Reply:
xradio reports that it is mesh point capable, however it is not.  So this forces the result to be false when dealing with the xradio_wlan driver.